Add English version README (#45)
gmt-china/china-geospatial-data: README-en.md

# China Geospatial Data for GMT

![GitHub repo size](https://img.shields.io/github/repo-size/gmt-china/china-geospatial-data)

This project contains China geospatial data that can be used in
[Generic Mapping Tools (GMT)](https://www.generic-mapping-tools.org/)

> -**Homepage**: https://github.com/gmt-china/china-geospatial-data
> -**Download**: https://github.com/gmt-china/china-geospatial-data/releases
> -**Use Instructions**: https://docs.gmt-china.org/latest/dataset-CN/ (In Chinese)
> 

Reviewer: ```suggestion
- **Homepage**: https://github.com/gmt-china/china-geospatial-data
- **Download**: https://github.com/gmt-china/china-geospatial-data/releases
- **Use Instructions**: https://docs.gmt-china.org/latest/dataset-CN/ (In Chinese)

```

Reviewer: Use Instructions -->> Instructions ?

## Data

### China Administrative Divisions

- [CN-border-La.gmt](CN-border-La.gmt): China's national borders, provincial borders,
  ten-segment lines, and South China Sea island data
- [CN-border-L1.gmt](CN-border-L1.gmt): China's national borders, ten-segment line,
  and South China Sea island data, excluding provincial data

> - [ten-dash-line.gmt](ten-dash-line.gmt): only contains ten-segment line data

Reviewer: ```suggestion
- [ten-dash-line.gmt](ten-dash-line.gmt): only ten-segment line data
```

> ### China mainland and neighboring blocks

Reviewer: ```suggestion
###  Active tectonic blocks in the China mainland and neighboring regions
```

> - [CN-block-L1.gmt](CN-block-L1.gmt): Level 1 block boundary data

Reviewer: ```suggestion
- [CN-block-L1.gmt](CN-block-L1.gmt): Level-1 block boundary data
```

> - [CN-block-L1-deduced.gmt](CN-block-L1-deduced.gmt): Dedudec Level 1 block boundary data

Reviewer: ```suggestion
- [CN-block-L1-deduced.gmt](CN-block-L1-deduced.gmt): Deduced Level-1 block boundary data
```

> - [CN-block-L2.gmt](CN-block-L2.gmt): Level 2 block boundary data

Reviewer: ```suggestion
- [CN-block-L2.gmt](CN-block-L2.gmt): Level-2 block boundary data
```

> ### China Fault

Reviewer: ```suggestion
### China Faults
```

> -[CN-faults.gmt](CN-faults.gmt): China faults

Reviewer: ```suggestion
- [CN-faults.gmt](CN-faults.gmt): China faults
```

> ### Geological Map of Mainland China and Adjacent Areas

Reviewer: See the difference between **region** and **area**: http://blog.sina.com.cn/s/blog_540316260100u07h.html
```suggestion
### Geological Map of the Mainland China and Adjacent Regions
```

> -[geo3al.gmt](geo3al.gmt): Geological map of mainland China and adjacent areas

Reviewer: ```suggestion
-[geo3al.gmt](geo3al.gmt): Geological map of the mainland China and adjacent regions
```

> -[geo3al.gmt](geo3al.gmt): Geological map of the mainland China and adjacent regions

Reviewer: ```suggestion
- [geo3al.gmt](geo3al.gmt): Geological map of the mainland China and adjacent regions
```

## Version history

If the **latest version** is different from the **current version**,
please download the latest version from the project homepage.

> -![GitHub release (latest by date)](https://img.shields.io/github/v/release/gmt-china/china-geospatial-data) (**Latest version**)
> -v0.3.0: 2021-06-25 (**Current Version**)
> -v0.2.0: 2021-06-08
> -v0.1.0: 2021-06-03

Reviewer: ```suggestion
- ![GitHub release (latest by date)](https://img.shields.io/github/v/release/gmt-china/china-geospatial-data) (**Latest version**)
- v0.3.0: 2021-06-25 (**Current Version**)
- v0.2.0: 2021-06-08
- v0.1.0: 2021-06-03
```

For detailed version history, please refer to [Changelog.md](Changelog.md).

## Maintainer

> The data in this project is provided by users and maintained by

Reviewer: ```suggestion
The data in this project is provided by users and maintained by the
```
